torchvision/prototype/features/_feature.py

```diff
@@ -42,7 +43,8 @@ def __new__(
         requires_grad: bool = False,
     ) -> _Feature:
         tensor = cls._to_tensor(data, dtype=dtype, device=device, requires_grad=requires_grad)
-        return tensor.as_subclass(_Feature)
+        output = tensor.as_subclass(_Feature)
+        return output
```

Comment:
Could you revert this?

```suggestion
        return tensor.as_subclass(_Feature)
```